### DISCUSSION TITLE
Difficult to see pull request status in Search results

### DISCUSSION THREAD

**Discussion by Marcono1234** [community/community#39593] [Accessibility] [answered]
### Select Topic Area

Product Feedback

### Body

When viewing the "Issues" tab on the Search results, the status of pull requests only seems to be indicated by its color. Especially for open and draft pull requests this makes it difficult to tell them apart.

This also differs from how the "Pull Requests" page shows it, which in addition to different colors also uses different icons to make it easier to determine the status of a pull request.

For comparison here is how the "Pull Requests" page (e.g. https://github.com/github/docs/pulls?q=is%3Apr) shows the status compared to how the "Search" page (e.g. https://github.com/github/docs/search?q=is%3Apr&type=issues) shows the status:

|Status|Pull Requests|Search: Issues|
|-|-|-|
|Draft|![draft icon](https://user-images.githubusercontent.com/11685886/202915731-54355a22-f017-4d62-80bc-0ee701e33774.png)|![draft icon](https://user-images.githubusercontent.com/11685886/202915512-93f3d6f4-10d8-47da-bdfc-6f715dcb829b.png)|
|Open|![open icon](https://user-images.githubusercontent.com/11685886/202915730-8c756efd-4b77-49f7-92bb-ebd0e4aab27d.png)|![open icon](https://user-images.githubusercontent.com/11685886/202915500-ccaf0127-9bdb-45d0-87bf-adf4f2dd0668.png)|
|Merged|![merged icon](https://user-images.githubusercontent.com/11685886/202915733-9094bfb7-9604-4a13-81b2-e9d59d5396ee.png)|![merged icon](https://user-images.githubusercontent.com/11685886/202915499-7ecaa477-b184-4c96-bf20-e8f45acac727.png)|
|Closed|![closed icon](https://user-images.githubusercontent.com/11685886/202915506-fd3c9d33-2e11-46a1-8e74-2757e586981a.png)|![closed icon](https://user-images.githubusercontent.com/11685886/202915497-6d59c75c-f839-4090-b9af-8f823d51c685.png)|

(at this scale you can more easily tell the icons apart (unless you are colorblind), but at the scale at which they are shown normally, this is not easily possible)

Note that this does not apply to issues displayed on the "Issues" tab of the "Search" page, for them different colors _and icons_ are used.

**Comment by cehfisher** [comment]
Hello @Marcono1234 - Thank you for providing this feedback! We will file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Comment by cehfisher** [comment, marked as answer]
Hello @Marcono1234 - great news, the accessibility team was able to get the icons updated for issues. Could you please confirm you see this fix on your end? Thanks again for taking the time to leave feedback!

**Reply by Marcono1234** [reply]
The difference is that it now uses distinct icons (in addition to different colors), right?
It is much better now, thanks!

**Reply by cehfisher** [reply]
Yes, the icons for `Search: Issues` are now the same as the `Pull Requests` icons which differ by both shape and color. I'm glad this improves your experience! I'll close this issue, but feel free to reopen it or start a new discussion post if you have any additional feedback.
